src/renderer.js

```diff
@@ -94,13 +94,8 @@ observeElement('.chat-input-area .ck-editor', function () {
         messageEl = getMessageElement(e.target);
         log('右键点击消息', messageEl);
         appended = false;
-    });
 
-    new MutationObserver(() => {
-        const qContextMenu = document.querySelector(".q-context-menu");
-        if (appended) {
-            return;
-        }
+        const qContextMenu = document.querySelector("div.q-context-menu:not(.more-menu)");
```

Comment:
Right click panel:
<img width="868" alt="图片" src="https://github.com/MUKAPP/LiteLoaderQQNT-DeepL/assets/91816094/4fb34eab-c53b-4530-aa15-6920c132fc53">

More items panel:
<img width="888" alt="图片" src="https://github.com/MUKAPP/LiteLoaderQQNT-DeepL/assets/91816094/39bda538-464c-40a5-8d6e-25745c7bc579">

This would resolve the misplace issue alone, but won't fix all problems on macOS altogether.